cosmos/mantrachain-testnet.json

```diff
@@ -0,0 +1,50 @@
+{
+  "chainId": "mantrachain-testnet-1",
+  "chainName": "MANTRA Chain Testnet",
+  "chainSymbolImageUrl": "https://github.com/chainapsis/keplr-chain-registry/blob/main/images/mantrachain-testnet/chain.png",
```

Comment:
@martinhalford Hello.

Could you fix `chainSymbolImageUrl`?
to `https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/mantrachain-testnet/chain.png`
from `https://github.com/chainapsis/keplr-chain-registry/blob/main/images/mantrachain-testnet/chain.png`

Reply:
@HeesungB - Update made and commit pushed.  Thanks!